setup/Sandbox/sandbox.rst

```diff
@@ -52,11 +60,23 @@ extension. The JupyterLab interface also supports plain text and Markdown files.
 
 To learn more about JupyterLab, visit the `JupyterLab Documentation`_.
 
-.. note::
-   Changes made to Jupyter notebooks in the DEA Sandbox may be automatically 
-   overwritten as part of the automatic update process. To avoid this, we
-   recommend advanced users use Git to clone a new copy of ``dea-notebook``
-   into the Sandbox (`see guide here`_).
+Some things to know about The Sandbox
```

Comment:
Something like "Important information about the Sandbox" might sound less informal